### DISCUSSION TITLE
Help, migrating from Cocoapods to SPM

### DISCUSSION THREAD

**Discussion by p-nicolaou** [firebase/firebase-ios-sdk#8804] [Q&A]
For our project, we have all CoreData related code in a seperate framework that we embed in our main target. That framework had a dependancy to FirebaseCrashlytics in our Podfile. So, in the General tab of our framework, I can see `Pods_XXCoreData.framework` -> **Do Not Embed** option. How can I achieve the same now that we are planning to migrate to SPM?

**Comment by paulb777** [comment]
I believe this can be done purely by defining the targets as you like in Xcode and pointing at libraries installed via SPM. 

For more detailed help, stackoverflow or a Swift Forum might be a better venue for help since there should be nothing Firebase specific about the setup.

**Reply by p-nicolaou** [reply]
Hello @paulb777 . It took a lot of research around (difficult to describe my issues) but I found an existing post on Swift forums describe exactly what I am facing. 
https://forums.swift.org/t/xcode-spm-integration-issue-xcode-bundles-frameworks-into-swift-package-frameworks-that-shouldnt-have-them/50119
